pub/dependabot-pub.gemspec

```diff
@@ -24,6 +24,8 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "dependabot-common", Dependabot::VERSION
 
+  spec.add_development_dependency "webrick", ">= 1.7"
```

Comment:
`webrick` was removed from the `ruby` `3.0` stdlib, so have to manually add it as a dependency.